tests/tests.h

```diff
@@ -45,15 +45,15 @@
 # endif
 
 # define INIT_BASE(var, l_stdout, o_stdout, l_stderr, o_stderr, p_name, init) \
-    sirinit var         = {0}; \
-    var.d_stdout.opts   = (o_stdout) > 0 ? o_stdout : SIRO_DEFAULT; \
-    var.d_stdout.levels = l_stdout; \
-    var.d_stderr.opts   = (o_stderr) > 0 ? o_stderr : SIRO_DEFAULT; \
-    var.d_stderr.levels = l_stderr; \
-    if (_sir_validstrnofail(p_name)) \
-        _sir_strncpy(var.name, SIR_MAXNAME, p_name, SIR_MAXNAME); \
-    bool var##_init = false; \
-    if (init) \
```

Comment:
This is weird. In the other macro, it left-aligned, now this one is right-aligned. We need to stick with one or the other... Let's go with LEFT.